cfme/tests/integration/test_cfme_auth.py

```diff
@@ -118,6 +118,7 @@ def user_obj(appliance, auth_user, user_type):
 @pytest.mark.uncollectif(lambda auth_user: not any([True for g in auth_user.groups or []
                                                    if 'evmgroup' in g.lower()]),
                          reason='No evm group available for user')
+@pytest.mark.browser_isolation
```

Comment:
maybe just put it on top of the file for all tests in this module?

Reply:
Yeah I think that should work fine - I was in the mentality of a fixture, and the scope not working right - but it should still operate at the function level just fine.